### ISSUE TITLE
"-" fieldtag generates valid Go but invalid Javascript client

### ISSUE THREAD

**Issue by hugows** [webrpc/webrpc#66]
A RIDL spec like the following:

```
message User
  - id: uint64
  - isAdmin: bool
  - createdAt: timestamp
  - email: string
  - emailVerified: bool
  - passwordHash: string
    + json = -
```

will generate the following Javascript, instead of ignoring the field:

```
export class User {
  constructor(_data) {
    this._data = {}
    if (_data) {
     ...
      this._data['emailVerified'] = _data['emailVerified']
      this._data['-'] = _data['-']
    }
  }
  ...
  get emailVerified() {
    return this._data['emailVerified']
  }
  set emailVerified(value) {
    this._data['emailVerified'] = value
  }
  get -() {
    return this._data['-']
  }
  set -(value) {
    this._data['-'] = value
  }
 ...
}
```

Replacing `json` with `go.tag.json` is a temporary workaround, IIUC.

**Comment by pkieltyka** [comment]
thanks for the note

**Comment by VojtechVitek** [comment]
Released in [v0.12.0](https://github.com/webrpc/webrpc/releases/tag/v0.12.0)
